hwilib/devices/bitbox02.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, msg: str):
         )
         msg += "m/49'/0'/<account'> for p2wpkh-p2sh; "
         msg += "m/84'/0'/<account'> for p2wpkh; "
+        mrs += "m/86'/0'/<account'> for p2tr;"
```

Comment:
most probably a typo?
```suggestion
        msg += "m/86'/0'/<account'> for p2tr;"
```